pkg/cli/mt_cert.go

```diff
@@ -140,3 +140,49 @@ If --overwrite is true, any existing files are overwritten.
 				"failed to generate tenant signing cert and key")
 		}),
 }
+
+// A mtCreateClientForTenantCert command generates a tenant scoped client certificate and stores it
+// in the cert directory under client.<username>.tenant-<tenant_id>.crt and key under client.<username>.tenant-<tenant_id>.key.
+var mtCreateClientForTenantCert = &cobra.Command{
+	Use:   "create-client-for-tenant --certs-dir=<path to cockroach certs dir> --ca-key=<path-to-ca-key> <username> <tenant_id>",
```

Comment:
not that i have any great idea, but i feel like `create-client-for-tenant` is going to be really easily confused with `create-tenant-client`. is there a helpful way that i should be thinking about this to avoid confusion?